vector/src/main/java/im/vector/app/AppStateHandler.kt

```diff
@@ -57,7 +56,7 @@ class AppStateHandler @Inject constructor(
         private val sessionDataSource: ActiveSessionDataSource,
         private val uiStateRepository: UiStateRepository,
         private val activeSessionHolder: ActiveSessionHolder
-) : LifecycleObserver {
+) : DefaultLifecycleObserver {
```

Comment:
nice clean up! might speed up the lifecycle KAPT processing a tiny bit as well :crossed_fingers: